app/controllers/app/application_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+module App
+  class ApplicationController < ActionController::Base
+
+  end
+end
```

Comment:
We usually do not use the `App` folder.

```suggestion
class ApplicationController < ActionController::Base
end
```